common/k8stest/util_rdma.go

```diff
@@ -317,3 +317,41 @@ func GetDevLinkName(node, iface string) (string, error) {
 	}
 	return "", nil
 }
+
+func DisableRdmaDeicesOnAllMayastorNodes() error {
+	workerNodes, err := ListIOEngineNodes()
+	if err != nil {
+		return err
+	}
+	for _, node := range workerNodes.Items {
+		logf.Log.Info("Disable rdma device", "Node", node.Name)
+		ifaceList := e2e_config.GetConfig().DisabledRdmaDevices
+		for _, iface := range ifaceList {
+			err := DisableRdmaOnNode(node.Name, iface)
+			if err != nil {
+				return err
+			}
+		}
+
+	}
+	return nil
+}
+
+func EnableRdmaDeicesOnAllMayastorNodes() error {
```

Comment:
typo `EnableRdmaDeicesOnAllMayastorNodes` -> `EnableRdmaDevicesOnAllMayastorNodes`

Reply:
Again not great but would this be better named as `RestoreConfiguredDisabledRdmaDevicesOnAllMayastorNodes`

Reply:
[Done](https://github.com/openebs/openebs-e2e/compare/97f6bb9f1285bb8f08fe736bb498948de52c0b3f..7a4e8d1886a4b1bebeba350d30dde9c35e2ec096)